package.json

```diff
@@ -37,7 +37,8 @@
     "rimraf": "~2.5.0",
     "tar": "~2.2.1",
     "tv4": "~1.2.7",
-    "swagger-tools": "~0.9.11"
+    "swagger-tools": "~0.9.11",
+    "moment": "~2.11.1"
```

Comment:
You've made this comment a few times, but this is the exact command I am using, and that is what the command did.  

Note that all of these packages are not alphabetized:
    "swagger-express-mw": "^0.6.0",
    "os-tmpdir": "~1.0.1",
    "rimraf": "~2.5.0",
    "tar": "~2.2.1",
    "tv4": "~1.2.7",
    "swagger-tools": "~0.9.11",
    "moment": "~2.11.1"
and:
   "nock": "~3.5.0"